src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -1,4 +1,5 @@
 require 'common/common'
+require 'membrane'
```

Comment:
You should not need to change anything about requirements, as far as I understand the change, it is just returning a hash. Membrane shouldn't be involved. When do you get any `unitialized_constant` issues? When executing specific tests?